cves/CVE-2017-9787.yml

```diff
@@ -73,10 +74,11 @@ fixes:
   - commit: 8b862f7cb2eaa6289ec9b36e7dc4a47521fc3bca
     note: Adds support for Spring proxies when chaining actions 2.5.x
 vccs:
-  - commit:
-    note:
-  - commit:
-    note:
+  - commit: 0c543aef318341ca9bd482e15f1637497b8a4dfd
+    note: No code was deleted or changed in the fix, so the lack of checking for
```

Comment:
You may want to dig into the fix a little more. Are we sure that is the correct fix commit?

Reply:
Or it may be a multi-commit fix

Reply:
Yes the fix is correct. It added a function for checking if you were using a proxy and what type it was. I reworded this to clarify that code was added, not deleted or changed.